app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -0,0 +1,177 @@
+class MiqCockpitWsWorker::Runner < MiqWorker::Runner
+  BINDING_ADDRESS = ENV['BINDING_ADDRESS'] || (Rails.env.production? ? nil : "0.0.0.0")
```

Comment:
OK, is it save to assume we can do 127.0.0.1? Or do we need to account for possibly IPv6 only systems?